pkg/controller/secrets/secrets.go

```diff
@@ -0,0 +1,87 @@
+package secrets
+
+import (
+	"io/ioutil"
+
+	wkl "github.com/openshift/windows-machine-config-operator/pkg/controller/wellknownlocations"
+	"github.com/pkg/errors"
+	"golang.org/x/crypto/ssh"
+	core "k8s.io/api/core/v1"
+	meta "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	userDataSecret    = "windows-user-data"
+	userDataNamespace = "openshift-machine-api"
+)
+
+// CreateSigner creates a signer using the private key from the privateKeyPath
+// TODO: As a part of https://issues.redhat.com/browse/WINC-316 , modify CreateSigner to take private key secret
+//  and return the signer
+func CreateSigner() (ssh.Signer, error) {
```

Comment:
Nit: from a purity perspective, this would be in a separate package called `signer`.